stable/pomerium/values.yaml

```diff
@@ -0,0 +1,130 @@
+# For detailed explanation of each of the configuration settings see
+# https://www.pomerium.io/docs/config-reference.htmls
+
+# settings that are shared by all services
+config:
+  # routes under this wildcard comain are handled by pomerium
+  domain: corp.pomerium.io
```

Comment:
@desimone you renamed it here but not in many other places where it is still named `rootDomain`.